crates/primitives/src/constants.rs

```diff
@@ -29,6 +29,6 @@ pub const TEST_PRIVATE_KEY: &str =
     "1212121212121212121212121212121212121212121212121212121212121212";
 
 pub const MAX_STATEDIFF_SIZE_COMMITMENT_THRESHOLD: u64 = 300 * 1024;
-pub const MAX_STATEDIFF_SIZE_PROOF_THRESHOLD: u64 = 400 * 1024;
+pub const MAX_STATEDIFF_SIZE_PROOF_THRESHOLD: u64 = 397000;
```

Comment:
Not sure how It is calculated but I took it from bitcoin-da MAX_TXBODY_SIZE
@eyusufatik told me to but it makes sense since there will be other parts of the tx so it should be smaller than 400kb for sure